src/main/java/duke/exception/EmptyDescriptionException.java

```diff
@@ -0,0 +1,6 @@
+package duke.exception;
+
+public class EmptyDescriptionException extends DukeException {
+
+    private static final long serialVersionUID = 4037951695361980047L;
```

Comment:
I'm not sure what is the serialVersionUID? Did you use some autogeneration tool to create this class?

Reply:
Yea I was using vscode and it recommended this. I removed it since it wasn't necessarily.